book/website/communication/citable/citable-cite.md

```diff
@@ -67,8 +48,7 @@ To learn more about how to cite physical samples and to check out examples, see
 ## Citing Software
 
 A software citation has a lot of the same elements as a data citation, described above, and are described in more detail in the [Software Citation Principles](https://www.force11.org/software-citation-principles).
-When using others software, it is vital to cite and attribute it properly.
-See also [How to Cite R and R Packages](https://ropensci.org/blog/2021/11/16/how-to-cite-r-and-r-packages/) for more information.
+Depending on the journal, you may also include a Software/Code Availability Statement. In such section, you can describe where the code developed during the research presented in the paper can be found, similar to how you would provide information about data availability.
```

Comment:
Do you have an example we could link to?

Reply:
Do you mean of the statements? or journals? From the issue there is this AGU guidance
https://www.agu.org/publish-with-agu/publish/author-resources/data-and-software-for-authors 
I also found this https://social-science-data-editors.github.io/guidance/Requested_information_dcas.html
and this one as a probable reference when talking about availability statements https://doi.org/10.1016/j.jclinepi.2022.03.003

Reply:
I meant example of the statements. I can imagine if I get to that section I would be wondering how a code availability statement looks like, and an example one would help. These examples are great!

Reply:
```suggestion
Depending on the journal, you may also include a Software/Code Availability Statement. In such section, you can describe where the code developed during the research presented in the paper can be found, similar to how you would provide information about data availability.
For an example you can look at the [Data and Code Availability Statements - Sample text](https://social-science-data-editors.github.io/guidance/Requested_information_dcas.html) or [Availability Statement Templates](https://www.agu.org/publish-with-agu/publish/author-resources/data-and-software-for-authors).
```